api/controllers/system.go

```diff
@@ -25,7 +25,9 @@ func SystemShow(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 }
 
 func SystemProcesses(rw http.ResponseWriter, r *http.Request) *httperr.Error {
-	ps, err := models.Provider().SystemProcesses()
+	all := r.URL.Query().Get("all")
+
+	ps, err := models.Provider().SystemProcesses(all == "true")
```

Comment:
Let's use an options struct here. See other provider methods like `BuildCreate` for an example.